packages/mui-material/src/Modal/Modal.test.js

```diff
@@ -896,4 +896,71 @@ describe('<Modal />', () => {
     expect(handleKeyDown).to.have.property('callCount', 1);
     expect(handleClose).to.have.property('callCount', 1);
   });
+
+  it('aria-hidden state managed properly with disablePortal', () => {
```

Comment:
This test and the one below are passing on the `master` branch, meaning they're not testing the correct changes. They should fail on `master` and pass on this branch.

Reply:
It passed because I didn't test if the modal content was accessible before. On master it isn't accessible (because aria-hidden gets applied to siblings of `body`).

Fixed